developer-guide/concepts-and-designs/sig-observability/kubesphere-monitoring.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The KubeSphere monitoring system comprises many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy  or create a few components within the namespace `kubesphere-monitoring-system`:
+The KubeSphere monitoring system comprises of many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy or create a few components within the namespace `kubesphere-monitoring-system`:
```

Comment:
I think `comprises` is good, or maybe `is comprised of `.

Reply:
should be `is comprised of`